packages/cli/src/hooks/analytics/session.spec.ts

```diff
@@ -0,0 +1,117 @@
+jest.mock('../../lib/config/config');
+
+import type {IConfig} from '@oclif/config';
+import {mocked} from 'ts-jest/utils';
+import {Config} from '../../lib/config/config';
+import check from './session';
+
+describe('session', () => {
+  const mockedConfig = mocked(Config, true);
+  const mockedConfigGet = jest.fn();
+  const mockedConfigSet = jest.fn();
+  const theDate = new Date('1997-08-29T12:00:00Z');
```

Comment:
I guess you need a non 0 epoch that's why?

Reply:
No. Just wanted to use the date during which Skynet became self-aware.